mslib/utils/mssautoplot.py

```diff
@@ -66,38 +63,14 @@ def load_from_ftml(filename):
     """
     _dirname, _name = os.path.split(filename)
     _fs = open_fs(_dirname)
-    datasource = _fs.open(_name)
-    wp_list = load_from_xml_data(datasource)
-    return wp_list
-
-
-def load_from_xml_data(datasource):
-    try:
-        doc = xml.dom.minidom.parse(datasource)
-    except xml.parsers.expat.ExpatError as ex:
-        raise SyntaxError(str(ex))
-
-    ft_el = doc.getElementsByTagName("FlightTrack")[0]
-
-    waypoints_list1 = []
-    waypoints_list2 = []
-    for wp_el in ft_el.getElementsByTagName("Waypoint"):
-
-        location = wp_el.getAttribute("location")
-        lat = float(wp_el.getAttribute("lat"))
-        lon = float(wp_el.getAttribute("lon"))
-        flightlevel = float(wp_el.getAttribute("flightlevel"))
-        comments = wp_el.getElementsByTagName("Comments")[0]
-        # If num of comments is 0(null comment), then return ''
-        if len(comments.childNodes):
-            comments = comments.childNodes[0].data.strip()
-        else:
-            comments = ''
-
-        waypoints_list1.append((lat, lon, flightlevel, location, comments))
-        waypoints_list2.append(ft.Waypoint(lat, lon, flightlevel, location, comments))
-        waypoints_list2[-1].utc_time = datetime.now()
-    return waypoints_list1, waypoints_list2
+    datasource = _fs.readtext(_name)
+    wp_list = ft.load_from_xml_data(datasource)
```

Comment:
important to postfix by the type?

Reply:
?
The naming of this functions is probaly not ideal.

Reply:
I meant more always adding "_list" to all these names, we don't do that regularly.